spec/fritzbox/smarthome/actor_spec.rb

```diff
@@ -71,6 +71,37 @@
       expect(actor.name).to                   eq "Sub-Etha Radio Transmitter"
       expect(actor.manufacturer).to           eq ""
       expect(actor.group_members).to          be nil
+
+      actor = actors.shift
+      expect(actor.class).to                  eq Fritzbox::Smarthome::Lightbulb
+      expect(actor.type).to                   eq :device
+      expect(actor.id).to                     eq "406"
+      expect(actor.ain).to                    eq "11111 2233445"
+      expect(actor.name).to                   eq "Flurlampe"
+      expect(actor.manufacturer).to           eq "AVM"
+      expect(actor.group_members).to          be nil
+    end
+  end
+
+  describe '.new_from_api' do
+    subject(:actor) { described_class.new_from_api(data) }
+
+    let(:data) { Hash.new }
+
+    context 'when setting the manufacturer' do
+      let!(:data) { { '@manufacturer' => 'Orang-Utan Klaus' } }
```

Comment:
@klausmeyer oh, just realized your first name. This is by no means a mockery, but rather a Helge Schneider reference. Happy to change this to something else

Reply:
No worries. I had to lookup the reference but then I got it :grin: